packages/near-membrane-base/src/membrane.ts

```diff
@@ -1455,23 +1501,46 @@ export function init(
             globalThisPointer,
         ] = hooks;
         foreignCallablePushTarget = callablePushTarget;
-        foreignCallableApply = foreignErrorControl(callableApply);
-        foreignCallableConstruct = foreignErrorControl(callableConstruct);
-        foreignCallableDefineProperty = foreignErrorControl(callableDefineProperty);
-        foreignCallableDeleteProperty = foreignErrorControl(callableDeleteProperty);
+        // traps utilities
+        foreignCallableApply = foreignErrorControl(
```

Comment:
these are onbound because they are coming from red, and will be used by blue to call into the red side.